src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -1683,7 +1683,8 @@ void M()
                 );
         }
 
-        [Fact, WorkItem(31370, "https://github.com/dotnet/roslyn/issues/31370")]
+        // PROTOTYPE(nullable-api): wants to change the type of the literal 2 to error type from int
```

Comment:
>PROTOTYPE(nullable-api): wants to change the type of the literal 2 to error type from int [](start = 11, length = 89)

Does this comment still apply?

Reply:
Yes. This is one of the tests that had errors, so I want to leave it here as I've already spend some time to determine what the actual issue is.

---
In reply to: [266131177](https://github.com/dotnet/roslyn/pull/34017#discussion_r266131177) [](ancestors = 266131177)